R-package/src/init.c

```diff
@@ -15,9 +15,17 @@ Check these declarations against the C/Fortran source code.
 */
 
 /* .Call calls */
+extern void XGBInitializeAltrepClass(DllInfo *info);
+extern SEXP XGDuplicate_R(SEXP);
+extern SEXP XGDuplicateAttrib(SEXP, SEXP);
+extern SEXP XGPointerEqComparison(SEXP, SEXP);
 extern SEXP XGBoosterTrainOneIter_R(SEXP, SEXP, SEXP, SEXP, SEXP);
 extern SEXP XGBoosterCreate_R(SEXP);
-extern SEXP XGBoosterCreateInEmptyObj_R(SEXP, SEXP);
+extern SEXP XGBoosterCopyInfoFromDMatrix(SEXP, SEXP);
```

Comment:
Let's keep the naming consistent for R-specific C functions, `_R()`.

Reply:
Renamed.